### PR TITLE
9093 - Implement common card component for homepage

### PR DESCRIPTION
**Pull request by anjenkin**
**High level description:**

Implement common card component for homepage

**Technical details:**

Created card components per the design system documentation - https://bahdigital.invisionapp.com/share/TWIAC4TP9VF#/screens/296077644

Implemented features that are used in the homepage mock

The common component lives in USAS /component/sharedComponents/commonCards but once it's used in the homepage tickets, storybook documentation is added and the component is at MVP level, it will be ported to the DTUI library.

Also a sample page includes the commons. localhost:3000/temp-page

The page is only visible in lower environments and can be deleted once the homepage cards are implemented.

**JIRA Ticket:**
[DEV-9093](https://federal-spending-transparency.atlassian.net/browse/DEV-9093)

**Mockup:**
Homepage mock - https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55/screen/630d278531b02d11b408e08e

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [n/a] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/homepage/hero/AnimatedHeading.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Hero.jsx
+ * CardHero.jsx
```
Comment (brianpetway) [inline src/js/components/homepage/hero/AnimatedHeading.jsx:2]: I don't think we need to rename this file here.
Was this a result of a find and replace?

File: src/js/components/homepage/hero/Hero.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Hero.jsx
+ * CardHero.jsx
```
Comment (brianpetway) [inline src/js/components/homepage/hero/Hero.jsx:2]: Same as above

**Comment by brianpetway** [comment]
It seems strange to me that the images in the inset variant do not have a border radius applied to them. But I guess that's a design question.

**Comment by brianpetway** [comment]
I don't see a version of the Card with no border and an icon instead of hero image, which we need for the Resources section.
Am I missing it somewhere?

**Comment by anjenkin** [comment]
@brianpetway 
Screen Shot 2022-09-09 at 3 06 18 PM

**Review by brianpetway — APPROVED** [review]

**Pull request merged**